.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## What does this PR do?
+
+## GitHub issue number?
```

Comment:
I personally like the GH issue to be at the top of the PR using one of the [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword), but It's totally a personal preference.